src/iuse.cpp

```diff
@@ -8445,8 +8445,14 @@ int iuse::camera(player *p, item *it, bool, point)
         }
 
         if (aim_point.x == p->posx() && aim_point.y == p->posy()) {
-            p->add_msg_if_player(_("You decide not to flash yourself."));
-            return 0;
+            if (query_yn(_("Are you sure you want to flash yourself?"))) {
+                p->add_msg_if_player(_("Great selfie. The flash blinds you."));
+                p->add_effect("blind", rng(5, 10));
```

Comment:
Either pull the change from this one, or add a check for welding/RM13/bionic glare-protection, please.  Should just be another commit or two, and push 'em to this branch: that'll auto-update the PR.